.github/workflows/gh-pages-test-deploy.yaml

```diff
@@ -0,0 +1,29 @@
+name: Test deployment
```

Comment:
Is this needed in `master`?

Reply:
yep, this checks if documentation could be successfully built